src/pages/HomePage/component/mytaskbox.tsx

```diff
@@ -0,0 +1,72 @@
+import { useQuery } from "react-query";
+import homeApi from "../../../libs/homeApi";
+
+const MytaskBox = () => {
+  const { isLoading, isError, data, error } = useQuery("qloKey", async () =>
+    homeApi.mytaskApi("user_id")
+  );
+
+  if (isLoading) {
+    return (
+      <div className="h-96 p-4 shadow-xl rounded-xl animate-pulse bg-gray-200"></div>
+    );
+  }
+  if (isError) {
+    console.log(error);
+    return (
+      <div className="h-96 p-4 bg-background-white shadow-xl rounded-xl">
+        <div className="p-2">
+          <h2 className="text-black font-bold text-xl">Tasks</h2>
+        </div>
+        <div className="h-4/5 bg-main rounded-xl grid place-content-center">
+          <h2 className="text-grey font-normal text-xl">Somthing wrong.</h2>
+        </div>
+      </div>
+    );
+  }
+  if (data == null) {
+    return (
+      <div className="h-96 p-4 bg-background-white shadow-xl rounded-xl">
```

Comment:
pls add key of element like

<div key={index} className="h-96 p-4 bg-background-white shadow-xl rounded-xl">